### PR TITLE
Move persistent in auth object

### PR DESCRIPTION
**Pull request by StanWarz**
Last PR for the [Asana ticket about remember me](https://app.asana.com/0/980131104682803/1156801986427609/f)

### CODE REVIEW

File: src/widgets/auth/views/loginViewComponent.jsx [hunk 1]
```diff
@@ -58,10 +58,15 @@ export const LoginForm = createForm({
 });
 
 export default class LoginView extends React.Component {
-    handleLogin = data => this.props.apiClient.loginWithPassword({
-        ...data,
-        ...pick(this.props, 'auth')
-    });
+    handleLogin = data => {
```
Comment (roxanemace) [inline src/widgets/auth/views/loginViewComponent.jsx:61]: Why did you do this change?

Comment (StanWarz) [reply src/widgets/auth/views/loginViewComponent.jsx:61]: I explained it on Slack. `persistent` has be passed inside `auth` field (`AuthOptions` type).
This code block also makes it overridable by editable options.

Comment (StanWarz) [reply src/widgets/auth/views/loginViewComponent.jsx:61]: If I do not merge both of the `auth` objects (one from the `component view form` and the other from `editable options`), then the one from `editable options` will superseed it entirely with the previous code.

**Comment by mehdi-reach5** [comment]
@StanWarz  Je teste ça ce matin, attend mon retour avant de merger stp

**Comment by mehdi-reach5** [comment]
> @StanWarz Je teste ça ce matin, attend mon retour avant de merger stp

C'est bon pour moi merci encore @StanWarz 👍 
Je jetterai un dernier coup d'oeil au HP après la release mais on a bien le comportement attendu sur la Sandbox.

**Pull request merged**